### DISCUSSION TITLE
Creating new Issues

### DISCUSSION THREAD

**Discussion by maneprajakta** [Praful932/Kitabe#29] [Ideas]
Hi @Praful932 shall we start raising the issues? We can discuss here before raising

**Comment by Praful932** [comment]
Sure, Here are some that we discussed earlier
- Add a book for later
- Done reading a book
- Better component hyperparameter search
- Rate from search
Also there are a few ui bugs after the last patch, not sure if we should create a single issue for all ui bugs or each
What are your thoughts @maneprajakta ?

**Reply by maneprajakta** [reply]
Also, we can add *Books Read by the user* section where the books that are rated by the user will be displayed along with the rating given.

Ya, there are UI bugs after the last patch. I think we can go for a single issue for all UI bugs.
Also, if you could check whether the search bar is working on the laptop. On my end, it's not working, but in the phone, it works perfectly.

Just attaching some UI bugs:
1.After user sign in the rate this book button shrinks in **Popular Among Us section**
![bug1](https://user-images.githubusercontent.com/57425995/107370709-57644980-6b09-11eb-8e29-378204f174bd.png)

2.In **Personalized Recommendation** the buttons are not aligned
![bug2](https://user-images.githubusercontent.com/57425995/107370840-82e73400-6b09-11eb-926e-3e2c2c0aca55.png)

3.Also in **Personalized Recommendation section** the button color does not change after rating a book.

**Reply by Praful932** [reply]
Search is working fine on my side.
Let's document all the UI bugs and create an issue, enhancement can be less priority
All are after user signs in - Fix rate this book button 
1. After user signs in, Page - Homepage(Popular among us section)
2. In personalized section, also add color change for button for validation(see other buttons)
3.  Explore page

Let me know If i missed something adding to yours, I'll proceed to create an issue after

**Reply by maneprajakta** [reply]
Ok.
yes, All points are included.
I think some deployment issue their site is showing application error.

**Reply by Praful932** [reply]
Which site?

**Reply by maneprajakta** [reply]
Deployed one, It's opening now. During the afternoon it was showing an error.

**Reply by Praful932** [reply]
Which deployed site tho? This projects?

**Reply by maneprajakta** [reply]
Yes  https://kitabe-app.herokuapp.com/

**Reply by Praful932** [reply]
Okay I'll create issue then 👀

**Reply by maneprajakta** [reply]
![my-passport-photo](https://user-images.githubusercontent.com/57425995/107533945-759f7780-6be5-11eb-8d0a-cdbc5ecb018b.jpg)

**Reply by Praful932** [reply]
Done :)

**Comment by maneprajakta** [comment]
Listing out some issues here so that we don't forget in future:
- [this](https://github.com/Praful932/Kitabe/blob/545cf9202cd5abcfa7343ea74ad5708ddedb3794/mainapp/views.py#L42) need to be changed to 152 for explore page.
- Adding tooltip for book & author name
- using the library for rendering book description

**Comment by Praful932** [comment]
Some urgent ones
- Reduce number of icons in navbar, possible reduce all features to one single icon with a nice dropdown?
- Popular Among Users uniform card size

Some suggested by users
- remove more details, and move it to - on click book card
- switch explore to order by tags, not same as genre rather we can keep multiple 1-3 tags per book and have them at the top, if clicked on one, the books would reorder

**Reply by maneprajakta** [reply]
Yep all seem valid to me :star:

**Comment by Praful932** [comment]
Any thoughts on what we should keep the icon to be?
Something like:
Library icon when clicked, a nice popup
Recommendations
Already read books
To read books
Explore

Wdyt

**Reply by maneprajakta** [reply]
Some thing like [this](https://codepen.io/maneprajakta/pen/MWJmMda) with library icon.

**Reply by Praful932** [reply]
This is nice. Do you mean this as a popup in the middle or as it is?

**Reply by maneprajakta** [reply]
Some thing like this  :
![navbar](https://user-images.githubusercontent.com/57425995/113528221-9977ac00-95dd-11eb-9148-ce19142026df.png)

**Reply by Praful932** [reply]
Ok, we will just need something for the logo, so user will know to click on it, since most of the stuff of our app is there

**Reply by maneprajakta** [reply]
Yep right .

**Comment by maneprajakta** [comment]
- For rate from search the star of ratYo can be set up same as we have throughout the website.

**Reply by Praful932** [reply]
Yep

**Comment by maneprajakta** [comment]
Another Bug :/ see  book name
![Screenshot from 2021-05-10 08-41-04](https://user-images.githubusercontent.com/57425995/117603376-83c34c80-b170-11eb-9665-ed2f12302f8a.png)

**Reply by Praful932** [reply]
Didn't get you

**Comment by Praful932** [comment]
- Add recommendations(your read) to dropdown
- Book desc from file instead of scraping
